test/RelayHub.test.ts

```diff
@@ -1574,7 +1577,34 @@ contract('RelayHub', function ([_, relayOwner, relayManager, relayWorker, incorr
     const gas = 4e6
 
     beforeEach(async function () {
-      relayHubInstance = await deployHub(penalizerMock)
+      relayHubInstance = await deployHub()
```

Comment:
i'm not a huge fan of deploying a new hub for each test, but i didn't want to completely disrupt the structure of what was already there